athena-jdbc/src/main/java/com/amazonaws/athena/connectors/jdbc/manager/JdbcRecordHandler.java

```diff
@@ -252,7 +251,14 @@ protected Extractor makeExtractor(Field field, ResultSet resultSet, Map<String,
             case FLOAT8:
                 return (Float8Extractor) (Object context, NullableFloat8Holder dst) ->
                 {
-                    dst.value = resultSet.getDouble(fieldName);
+                    try {
+                        dst.value = resultSet.getDouble(fieldName);
+                    }
+                    catch (java.sql.SQLException ex) {
```

Comment:
Is the exception thrown the generic SqlException? or can we make this more specific?

Reply:
The actual exception thrown is from Postgres and it extends from SqlException, but since we are in athena-jdbc, we don't want to introduce dependencies in postgres, so we catch SqlException instead.

Reply:
I'm fine with this then, thanks for clarifying.